plugin/teksi_wastewater/teksi_wastewater_plugin.py

```diff
@@ -154,6 +154,10 @@ def initGui(self):
         self.wastewater_networkelement_layer_notifier = TwwLayerNotifier(
             self.iface.mainWindow(), ["vw_wastewater_node", "vw_tww_reach"]
         )
+        self.wastewater_structure_layer_notifier = TwwLayerNotifier(
+            self.iface.mainWindow(), ["vw_wastewater_structure"]
```

Comment:
This is not working (at least for me the slot never get called) because in the project there is no layer named `vw_wastewater_structure`
If you need only a project opened/closed slot you could use `_wastewater_networkelement_layer_available_changed` without making a new listener

Reply:
I will alter it to vw_tww_wastewater_structure